monitoring/grafana/dashboards/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(CMAKE_INSTALL_GRAFANA_DASHBOARDS "${CMAKE_INSTALL_SYSCONFDIR}/grafana/dashboards/ceph-dashboard"
```

Comment:
i think `CMAKE_INSTALL_*` paths are set and/or use by cmake and its module(s), in this case, it's not a well known cmake variables out side of ceph, so i guess we could just use `CEPH_GRAFANA_DASHBOARDS_DIR` or `CEPH_GRAFANA_DASHBOARDS_PATH` here.